### ISSUE TITLE
Wrong values for the function getExtractedText

### ISSUE THREAD

**Issue by orwel01** [CollaboraOnline/online#5737]
**Describe the bug**
I am now using Collabora Office as my primary tool for editing ODT, DOC files on the Android Phone. I also use another great app - MyPhoneExplorer to sync + write on Android phone directly from PC.
There is a Textinpunt function (SHIFT+F7) on MyPhoneExplorer Desktop (WIN10), which is not working well with the Collabora Office app. I got an answer from the developer of the MPE app:
_I reproduced the issue and i can confirm the problem.
But the issue can be fixed only from the developers of Collabora Office. MyPhoneExplorer does use - as every other soft keyboard too - the "InputConnection" API and it seems your app does return the wrong values for the function getExtractedText (only 2 space chars)
https://developer.android.com/reference/android/view/inputmethod/InputConnection#getExtractedText(android.view.inputmethod.ExtractedTextRequest,%20int)_
(see at https://www.fjsoft.at/forum/viewtopic.php?t=36556).

**To Reproduce**
1. Install MPE app (https://www.fjsoft.at/en/) - both Client and Desktop
2. Allow USB debugging on Android phone
3. Connect Android phone via USB cable, start MPE desktop
4. After connection use Textinpunt function (SHIFT+F7).
5. Reopen some file  or Copy-paste some text within the  Textinpunt function at MPE desktop as described below at Additional context.

**Expected behavior**
The text should not disappear.

**Actual behavior**
The text in PC window (MPE textinput) disapears.

**Desktop (please complete the following information)**
 - OS: [Win10]
 - Browser: [Firefox]

**Smartphone (please complete the following information)**
 - Device: [Realme GT Master]
 - OS: [Android 12]
 - Browser: [Opera, Firefox]

**Additional context**
The Textinput function of MPE (desktop) works well by opening a new file in Collabora Office (at android phone), this means you can write from Textinput (desktop) directly to the file and you see the text. But after closing the file (on Android phone) and reopening it, the Textinput window (desktop) does not show the written text anymore, neither if refresh function is used.
The same behavior can be noticed if you does not close the file (this means, you edit it very first time as a blank document in Collabora Office) and you paste some text to file - it does not matter if you copy-paste it directly on the Android phone (without using Textinput of MPE, e. g. from another app or you copy-paste some text directly in the Textinput window of MPE (desktop).
I think this can be a bug in MPE as this problem does not occur e. g. by using WPS Office android app.

**Comment by Darshan-upadhyay1110** [comment]
hii @orwel01 is it still the issue? I recommend to use latest version for smoother experience. Feel free to comment as this is too old i am closing this for now.

Thanks

**Comment by orwel01** [comment]
Yes, the bug is still present.

**Comment by orwel01** [comment]
Amendment: .txt file is not affected, but .docx, .doc, .odt files are

**Comment by Darshan-upadhyay1110** [comment]
Oh sorry to hear that @orwel01. Will look into this...

**Comment by Ezinnem** [comment]
cc @Minion3665

**Comment by Ezinnem** [comment]
@orwel01 
could you please recheck whether this issue is still reproducible with a recent version?


Feel free to comment if you can still reproduce

**Comment by orwel01** [comment]
Yes, the bug is still present (tested on .docx, .doc, .odt files located on android device). .txt is not affected,
